doc/getting_started.md

```diff
@@ -16,28 +16,17 @@ How to install and initialize LXD:
 
 :diataxis:Install LXD </installing>
 :diataxis:Initialize LXD </howto/initialize>
-:diataxis:Manage the snap </howto/snap>
 ```
 
 ```{only} diataxis
-How to enable access to the UI and the documentation:
+How to access the UI and the local build of the documentation:
```

Comment:
"build" here feels confusing. Maybe "local copy" or "bundled documentation" or "offline copy of the documentation"?

It's good to mention that it's local as it is useful for air-gapped environments.